website/source/docs/commands/push.html.md

```diff
@@ -44,6 +44,11 @@ configuration using the options below.
   `hashicorp/precise64`, which follows the form `<username>/<buildname>`. This
   must be specified here or in your template.
 
+- `-private` - A comma-separated list of variables that should be marked as
```

Comment:
I think it's worth being extra clear that this is a list of variable names, and not values. Perhaps in the example you could add `-var 'supersecretpassword1=mypassword'` or something?